d1: Update query-d1-from-python-workers.md (#13751)
cloudflare/cloudflare-docs: content/d1/examples/query-d1-from-python-workers.md

---
type: example

> summary: Learn how to query D1 from a Python Worker

Reviewer: I think this summary can stay as `Query D1 from a Python Worker`, it's consistent with the other examples.

Author: It seems to render really weirdly - it looks like a mistake/rendering issue since it's so close to the actual title:

<img width="849" alt="image" src="https://github.com/cloudflare/cloudflare-docs/assets/18544/e7a28c6b-f217-4948-9ee1-9f2b3af70f98">

Author: With the PR as-is:

<img width="1041" alt="image" src="https://github.com/cloudflare/cloudflare-docs/assets/18544/069a64f2-566b-452c-9e3b-ce63486d91a0">

Reviewer: Sorry, I checked locally and it's fine, I confused the `title` with the `summary` tag.

2. Refer to the [Python language guide](/workers/languages/python/) to understand how Python support works on the Workers platform.
3. Have basic familiarity with the Python language.

> If you are new to Cloudflare Workers, we recommend [following the getting started guide](/workers/get-started/guide/) first before coming back to this example.

Reviewer: ```suggestion
If you are new to Cloudflare Workers, refer to the [Get started guide](/workers/get-started/guide/) first before continuing with this example.
```
